tests/database/factories/EpisodeFactory.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+use Faker\Generator as Faker;
+
+/** @var Illuminate\Database\Eloquent\Factory $factory */
+$factory->define(\Tests\Utils\Models\Episode::class, static fn (Faker $faker): array => [
+    'title' => $faker->title,
+    'schedule_at' => $faker->date(),
```

Comment:
```suggestion
    'schedule_at' => $faker->randomElement([$faker->date(), null]),
```

Reply:
removed try/catch

` if (!$value) {
            return $builder;
        }
        $scopes = $this->directiveArgValue('names', $this->nodeName());

        return $builder->withoutGlobalScopes($scopes);`

Reply:
Please reread the suggestion in question.